### DISCUSSION TITLE
Help us develop the application and become a tester!

### DISCUSSION THREAD

**Discussion by Julien-cpsn** [Julien-cpsn/ATAC#53] [Announcements]
Hello everyone!

First, thanks to all of you for using and liking ATAC, it makes me very proud :)

When implementing new big features I often ask myself if I made everything alright and that I broke nothing. Now that we are a little community, a great help for me would be to have some people to help me test the features before they come out. 

2-3 people or even 1 person would be very great!

The tester will not be bothered very long, like 1 time every one or two week. And the "tests" should not be that deep, just to provide some feedback and avoid bugs that I would not have thought of.

If you are interested, leave a comment or contact me as you want

Thanks for reading this and have a great day ✨

EDIT: testers will be tagged as contributors

**Comment by Julien-cpsn** [comment]
My bad I forgot to turn the answers on -'

**Comment by NachoNievaG** [comment]
I believe you could count me in 😄
I don't know if there could be any other way to communicate more actively.

PS: Yay! 1K stargazers

**Reply by Julien-cpsn** [reply]
> I believe you could count me in 😄

Incredible, thanks man!!

> I don't know if there could be any other way to communicate more actively.

I think I did not understand this one 😂
If you speak about how can we communicate on the "tests", discord would be great but that is as you like

> PS: Yay! 1K stargazers

That is totally amazing yes! Although, I don't know many active user are they 🤔

**Comment by fnuttens** [comment]
I’m also interested in testing atac 🙂

**Comment by Julien-cpsn** [comment]
Hello @fnuttens and @NachoNievaG 

I am really busy for now but I really appreciate your engagement!!

What about creating a small discord?

Have a great day

**Reply by NachoNievaG** [reply]
For sure! Count me in

**Reply by Julien-cpsn** [reply]
@fnuttens  @NachoNievaG there you go https://discord.gg/Pd5hk6YT

**Comment by HarshitShukla-dev** [comment]
I am interested. ✌️

**Reply by Julien-cpsn** [reply]
Great!
You can join with that link https://discord.gg/Pd5hk6YT

**Reply by HarshitShukla-dev** [reply]
Umm... the invitation link has expired. Can you please share a new one.

**Reply by Julien-cpsn** [reply]
> Umm... the invitation link has expired. Can you please share a new one.

Excuse me! Here's another one https://discord.com/invite/WpTGyvFWFa
